src/CommunityVoices/App/Api/View/DisplayError.php

```diff
@@ -23,4 +23,14 @@ protected function getError($request)
 
         return $response;
     }
+    protected function getAllErrors($request)
+    {
+        $errors = explode('[]',str_replace('\n','',file_get_contents('/var/www/html/log/access.log')));
```

Comment:
This is certainly a creative way of doing this, but for me many of the items in the file still contain a leading newline character.  This happens because you can only use the newline character in PHP within double quotes.  Right now, it gets interpreted as the actual sequence of `\` followed by `n`.

I wonder if it might be easier to just only split by newline character?